content/sensu-enterprise/2.6/guides/using-snow.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "Using the ServiceNow Integration with Sensu Enterprise"
+linkTitle: "ServiceNow Integration"
+product: "Sensu Enterprise"
+version: "2.6"
+weight: 1
+menu:
+ sensu-enterprise-2.6:
+   parent: guides
+---
+
+In this guide, we'll cover configuring and using the Sensu Enterprise ServiceNow (SNOW) integration.
+
+# Prerequisites
+
+Before diving into this guide, we recommend having the following components ready:
+
+- A working Sensu Enterprise deployment
+- Working SNOW Instance (you can sign up for a test instance at [developer.servicenow.com][1])
+- SNOW credentials
+
+## ServiceNow Integration Basics
+
+Sensu Enterprise comes with a built-in integration for handling automatic registration and deregistration Sensu clients in SNOW. The integration also allows for creating incidents and events from Sensu incidents and events. We'll start off by configuring the integration, then move on to clients and checks. We'll finish by testing our integration and creating some test incidents to ensure that we've configured everything to our liking.
+
+_NOTE: Given SNOW's highly customizable nature, we'll only cover a small portion of what can be configured. For any in-depth discussion about the SNOW integration configuration, we encourage you to make use of the `#enterprise` channel in the [Sensu Community Slack][2]_
+
+### Integration Configuration
+
+Below, we have an example configuration that would typically live under `/etc/sensu/conf.d/servicenow.json`. 
+
+{{< highlight json >}}{
+  "servicenow": {
+    "host": "devXXXXXX.service-now.com",
+    "user": "test_user",
+    "password": "XXXXXXXXXXXXXXXX",
+    "create_cmdb_ci": true,
+    "cmdb_ci_table": "cmdb_ci_server",
+    "incident_management": true,
+    "incident_table": "incident",
+    "timeout": 10
+  }
+}{{< /highlight >}}
+
+Let's take a moment to walk through what we're specifying in that config file:
+
+* We three required attributes: `"host"`, `"user"`, and `"password"`
+* We then provide five additional attributes:
+  * `"create_cmdb_ci"` - this will create the configuration items automatically for Sensu clients. By default, this is `true`
```

Comment:
Maybe change the language around describing the default so the can all be consistent. line below uses `here is` vs `this is` for all the other lines.